s3fs/tests/test_s3fs.py

```diff
@@ -1092,14 +1140,15 @@ def test_versions(s3):
     with s3.open(versioned_file, 'wb') as fo:
         fo.write(b'2')
     versions = s3.object_version_info(versioned_file)
-    assert len(versions) == 2
+    version_ids = [version['VersionId'] for version in versions]
+    assert len(version_ids) == 2
 
     with s3.open(versioned_file) as fo:
-        assert fo.version_id == '1'
+        assert fo.version_id == version_ids[1]
```

Comment:
I'm sure I already wrote this fix too!

Reply:
Yep, I think I saw that in another PR, (but not in master, I don't think).